ecl/eclagent/eclagent.cpp

```diff
@@ -2502,6 +2502,8 @@ bool EclAgent::checkPersistUptoDate(IRuntimeWorkflowItem & item, const char * lo
         errText.appendf("Rebuilding PERSIST('%s'): Saved CRC isn't present", logicalName);
```

Comment:
Not directly related to this PR but the above constances (-2) should use the IWUResult constants defined in workunit.hpp, i.e. ResultSequencePersist in place of -2.
(The Roxie implementation of checkPersistUptoDate does)